### DISCUSSION TITLE
Does this work with Github Enterprise?

### DISCUSSION THREAD

**Discussion by robertklaus-bosch** [metcalfc/changelog-generator#156] [Q&A]
I would like to use this action on our hosted Github Enterprise instance. It does not work as expected as I get 'Error: Not Found' 
While looking for a possible cause, I stumbled over the line here:
https://github.com/metcalfc/changelog-generator/blob/4d3002ba1b6869096c5a2adb4b8c21a7ee39939a/changelog.sh#L32

Does someone use this action successfully with Github Enterprise (where the url will not be https://github.com but a custom one)?

**Comment by metcalfc** [comment]
It would be possible to pull that into a variable. I don’t have access to a ghe instance to test it.

**Reply by robertklaus-bosch** [reply]
I think it is possible to use an unreleased action from a branch at least the docs state:

> To use a specific action version, users can configure their GitHub Actions workflow to target a tag, a commit's SHA, or a branch named for a release.

So basically I could try it with our instance to see if it works.

Edit:
`GITHUB_SERVER_URL` is provided as environment variable - maybe this already does the trick?
